codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -135,6 +134,27 @@ public void onPriorityTreeParentChanging(Http2Stream stream, Http2Stream newPare
         });
     }
 
+    /**
+     * {@inheritDoc}
+     * <p>
+     * Any queued {@link FlowControlled} objects will be sent.
+     */
+    @Override
+    public void channelHandlerContext(ChannelHandlerContext ctx) throws Http2Exception {
```

Comment:
Why does the remote flow controller need the context at all? It never writes to the context directly. Maybe we should get rid of the context and just have the encoder worry about it?